src/parser/jobs/drk/modules/Delirium.tsx

```diff
@@ -92,7 +95,13 @@ export default class Delirium extends Module {
 		if (actionId === ACTIONS.DELIRIUM.id) {
 			const delirium = new DeliriumState(event.timestamp)
 			const fightTimeRemaining = this.parser.fight.end_time - event.timestamp
-			delirium.isRushing = DELIRIUM_DURATION >= fightTimeRemaining
+			if ( DELIRIUM_DURATION >= fightTimeRemaining ) {
```

Comment:
starting to see some duplication; wonder if it's something that can be abstracted away in future